GettingStarted.md

```diff
@@ -48,24 +48,24 @@ Include the Oboe header:
 
     #include <oboe/Oboe.h>
 
-Streams are built using an `OboeStreamBuilder`. Create one like this:
+Streams are built using an `StreamBuilder`. Create one like this:
```

Comment:
c/an/a/

Reply:
Are the class names Stream and StreamBuilder too generic? Is there a danger of colliding if developers are using multiple namespaces?

Reply:
There would only be a danger if they're using non-qualified namespaces e.g. 

`using namespace oboe; 
Stream stream;`

But yes, I do think `Stream` is too generic. `AudioStream`, `AudioStreamBuilder` etc would be more readable. Also if we ever support MIDI we could have a `MidiStream`. If you agree I'll rename.